analyses/copy_number_consensus_call/src/scripts/compare_variant_calling_updated.py

```diff
@@ -0,0 +1,257 @@
+parser.add_argument('--manta_cnvkit', required=True,
+                    help='path of the output consensus between manta and cnvkit')
+parser.add_argument('--manta_freec', required=True,
+                    help='path of the output consensus between manta and freec')
+parser.add_argument('--cnvkit_freec', required=True,
+                    help='path of the output consensus between cnvkit and freec')
+parser.add_argument('--sample', default='no_sample',
+                    help='sample name to use in the file')
+
+args = parser.parse_args()
+
+## Put the input and output file paths into their own lists that is to be iterated over
+## This order is important
+list_of_files = [args.manta, args.cnvkit, args.freec]
+list_of_output_files = [args.manta_cnvkit, args.manta_freec, args.cnvkit_freec]
+
+## Create a list to store the content of the 3 caller CNV files
+list_of_list = []
+
+
+## Loop through the file paths and read in their contents to store in a list
+for i in list_of_files:
+
+    ## Check to see if the CNVs bed file is empty, if so, make an empty variable
+    if os.stat(i).st_size == 0:
+        fin_input_content = []
+
+    ## If the file has content, then load in the content
+    else:
+        with open(i) as file:
+
+            ## Strip the trailing spaces for each line
+            stripped_content = [i.rstrip() for i in file.readlines()]
+
+            ## Split each line up by any white space between columns
+            fin_input_content = [re.split('\s+',i) for i in stripped_content]
+
+            ## If the 1st column has '1' instead of 'chr1' for the chromosome number, add in 'chr'
+            if fin_input_content[0][0].find('chr') == -1:
+                for c,chromo in enumerate(fin_input_content):
+                    fin_input_content[c][0] = 'chr' + fin_input_content[c][0]
+
+    ## Add the content to a list
+    list_of_list += [fin_input_content]
+
+
+## Make a list of iteraby index for the MAIN for-loop below
+## This give a list as followed: [0,1,2]
+list_index = list(range(0,len(list_of_list)))
+
+
+## We have 3 items in list_of_list. Each item is the content of an input file
+## Make a list to store the final files
+fin_list =[]
+
+## Loop through list_index
+for j, jval in enumerate(list_index):
+    for k,kval in enumerate(list_index[j+1:]):
+
+        ## Compare 1st and 2nd file from list_of_list
+        ## Then compare the 1st and 3rd
+        ## Then compare the 2nd and 3rd
+        list1 = list_of_list[jval]
+        list2 = list_of_list[kval]
+
+        ## Define a variable to store the list of consensus CNVs
+        consensus_list = []
+
+        ## For every CNVs in list1, we look through the ENTIRETY of list2 and perform actions if 2 CNVs overlaps
+        ## Could probably speed this up by using a dictionary.
+        for m in list1:
+
+            ## Assign the start_pos, end_pos, and copy number of list1 to variables
+            start_list1 = int(m[1])
```

Comment:
I am suggesting we do this for clarity and parallelism
```suggestion
            chr_list1 = m[0]
            start_list1 = int(m[1])
```